tasks.py

```diff
@@ -239,7 +238,40 @@ def _should_ignore(path):
     run(["kubeval"] + kube_yamls)
 
 
-@task(pre=[kubeval])
+@task()
+def update_from_templates(ctx):
+    """
+    Update kube yaml merges from templates
+    :param Context ctx:
+    """
+    devops.tasks.update_from_templates(ctx)
+
+
+@task()
+def _update_from_templates_hook(ctx):
```

Comment:
Any specific reason for `_` prefix?

Reply:
Yes, it's only intended to be used from the `pre_commit` task. For all normal use, the public `update_from_templates` should be used. This `_` version reaches out to git to check the status of files in order to be able to raise an exception in case new files are created that are not yet in version control, as the pre-commit doesn't stop a commit if new files are added as part of the hook, only if existing ones are modified.